.github/workflows/acceptance.yml

```
@@ -72,7 +72,7 @@ jobs:
 
   acceptance:
     # TODO(joshuarli): Convert to py3 with snapshots. See other TODO as well.
-    name: python2.7 acceptance
+    name: python acceptance
```

So we will need to update GH branch settings to reflect the new name. We should drop `python` as it's not relevant.

```suggestion
    name: acceptance
```